App.Stocks/Services/ValidateServices.cs

```diff
@@ -0,0 +1,48 @@
+﻿using App.Stocks.Models;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Text;
+
+namespace App.Stocks.Services
+{
+    public class ValidateServices : IValidateServices
+    {
+
+        public void ValidateCompany(Company company, bool IsOpenStocks = true)
+        {
+            if (company == null) throw new HttpListenerException(400, "Company doesn't exist");
```

Comment:
each statement should appear on new line, according to Microsoft guidelines. Please, place throwing exception on new line